subsys/bluetooth/host/conn_internal.h

```diff
@@ -74,13 +74,21 @@ struct bt_conn_sco {
 };
 #endif
 
-typedef void (*bt_conn_tx_cb_t)(struct bt_conn *conn);
+typedef void (*bt_conn_tx_cb_t)(struct bt_conn *conn, void *context);
 
 struct bt_conn_tx {
 	sys_snode_t node;
 	bt_conn_tx_cb_t cb;
+	void *context;
```

Comment:
The convention used elsewhere in the Bluetooth (and Networking) code is to call such context variables that get passed to callbacks user_data.

Reply:
That said, I kind of like "context" since that's what this is in practice. It's also a shorter name :)